src/services/campaignConfig/campaignConfig.ts

```diff
@@ -0,0 +1,54 @@
+import { IS_MOCK } from "../../config";
+import { CampaignConfig, ConfigHashes } from "../../types";
+import { fetchWithValidator, ValidationError } from "../helpers";
+import * as Sentry from "sentry-expo";
+
+export class CampaignConfigError extends Error {
+  constructor(message: string) {
+    super(message);
+    this.name = "CampaignConfigError";
+  }
+}
+
+const liveGetCampaignConfig = async (
+  token: string,
+  endpoint: string,
+  configHashes: ConfigHashes
+): Promise<CampaignConfig> => {
+  try {
+    const response = await fetchWithValidator(
+      CampaignConfig,
+      `${endpoint}/client-config`,
+      {
+        method: "POST",
+        headers: {
+          Authorization: token
+        },
+        body: JSON.stringify(configHashes)
+      }
+    );
+    return response;
+  } catch (e) {
+    if (e instanceof ValidationError) {
+      Sentry.captureException(e);
+    }
+    throw new CampaignConfigError(e.message);
+  }
+};
+
+const mockGetCampaignConfig = async (
+  _token: string,
+  _endpoint: string,
+  configHashes: ConfigHashes
+): Promise<CampaignConfig> => {
+  return {
+    features: {
+      minAppBinaryVersion: "3.0.0",
+      minAppBuildVersion: 0
+    }
```

Comment:
we will be having the existing features in campaign config as well right? i.e. transactionGrouping, id, flowType. 
or for now we are just using this endpoint to fetch the app versions?

Reply:
Yep we will add them in once we actually switch over, so yea you're right, just an endpoint to fetch the app versions